omnibus/config/projects/agent.rb

```diff
@@ -191,3 +191,8 @@
 
 exclude '\.git*'
 exclude 'bundler\/git'
+
+if linux?
+  strip_build true
+  debug_path ".debug"  # the srip symbols will be in here
```

Comment:
You don't reference this `.debug` directory anywhere else? Just making sure this is intended.

Reply:
Yes, this is intended, the `.debug` path is created by the stripper by `omnibus-ruby`. I'll add a clarifying note.